core/templates/domain/feedback_message/ThreadMessageSummary.model.spec.ts

```diff
@@ -13,27 +13,19 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for ThreadMessageSummaryObjectFactory.
+ * @fileoverview Unit tests for ThreadMessageSummary-model.ts.
  */
 
-import { TestBed } from '@angular/core/testing';
 
-import { ThreadMessageSummaryObjectFactory } from
-  'domain/feedback_message/ThreadMessageSummaryObjectFactory';
+import { ThreadMessageSummary } from
+  'domain/feedback_message/ThreadMessageSummary.model';
 
 describe('Thread message summary object factory', () => {
-  let threadMessageSummaryObjectFactory: ThreadMessageSummaryObjectFactory;
-
-  beforeEach(() => {
-    threadMessageSummaryObjectFactory = TestBed.get(
-      ThreadMessageSummaryObjectFactory);
-  });
-
   describe('.createNew', () => {
```

Comment:
Edit this, as the function doesn't exist anymore

Reply:
done